pyproject.toml

```diff
@@ -7,18 +7,27 @@ build-backend = "flit_core.buildapi"
 [project]
```

Comment:
I have an impression you have to lint up the poetry file for every PR. Can we just make a small , fast-landing PR to do this and we are done  ?

(Other option is to have a stacked PR, i.e. the next PR is targeting the branch of the previous one)